templates/workspaces/base/terraform/azure-monitor/azure-monitor.tf

```diff
@@ -79,6 +79,7 @@ resource "azurerm_monitor_private_link_scoped_service" "ampls_app_insights" {
 }
 
 resource "azurerm_private_endpoint" "azure_monitor_private_endpoint" {
+  count               = 0 # Remove with https://github.com/microsoft/AzureTRE/issues/2357
```

Comment:
how is this related to the issue?
should we mention anything about this block there?

Reply:
there 2 things addressed here: the problem with the PE which is happening for some time, and a newer one about the provider (checkpoint) which I also tried to address here since I couldn't get PRs to pass E2Es.